### PR TITLE
Mapzen isochrones integrated

### PR DESCRIPTION
**Pull request by ethervoid**
- Changed cdb_ischrones to use the [new isochrones API](https://mapzen.com/documentation/mobility/isochrone/api-reference/) released by Mapzen

### CODE REVIEW

File: server/extension/cdb_dataservices_server.control [hunk 1]
```diff
@@ -1,5 +1,5 @@
 comment = 'CartoDB dataservices server extension'
-default_version = '0.15.1'
+default_version = '0.16'
```
Comment (rafatower) [inline server/extension/cdb_dataservices_server.control]: not to be picky but.. `0.16.0`?

**Comment by rafatower** [comment]
made a couple of minor comments, other than that looks nice!

File: server/extension/cdb_dataservices_server.control (hunk 1 above)
Comment (ethervoid) [inline server/extension/cdb_dataservices_server.control]: Yep, you're right :)

**Comment by ethervoid** [comment]
Maybe we could use it as our Mapzen isochrone function in productions

// @javisantana

**Comment by javisantana** [comment]
could you ask them?

**Comment by ethervoid** [comment]
Sure, I'll let them know to check if they have any inconveniences to have this in production

**Comment by ethervoid** [comment]
Merged and checked against the API they published for the stable version [here](https://mapzen.com/documentation/mobility/isochrone/api-reference/)

There are two new options called `denoise` and `generalize` that are not implemented util we value them as useful :)

**Comment by ethervoid** [comment]
Please @rafatower could you check this please :)

File: server/extension/sql/80_isolines_helper.sql [hunk 2]
```diff
@@ -57,11 +57,9 @@ RETURNS SETOF cdb_dataservices_server.isoline AS $$
     quota_service.increment_total_service_use()
 $$ LANGUAGE plpythonu SECURITY DEFINER;
 
-
 CREATE OR REPLACE FUNCTION cdb_dataservices_server._cdb_mapzen_isolines(
```
Comment (rafatower) [inline server/extension/sql/80_isolines_helper.sql]: If I understand it correctly, I'd rename this to `_cdb_mapzen_isodistances`

**Comment by rafatower** [comment]
just a minor change request

**Pull request closed without merging**